extensions/wikia/EmergencyBroadcastSystem/EmergencyBroadcastSystemController.class.php

```diff
@@ -3,4 +3,40 @@ class EmergencyBroadcastSystemController extends WikiaController {
 	public function index( ) {
 		$this->response->setVal( 'nonPortableCount', '3' ); // Temporary number for testing
 	}
+
+	public static function saveUserResponse( $val ) {
+		global $wgUser;
+		if ($val === null) {
+			// invalid call, do nothing
+		} elseif ($val === 0) { // no
+			// if user clicks no, set ebs_response to 0
+			$wgUser->setOption( 'ebs_response', 0 );
```

Comment:
Maybe create a static with "ebs_response" - to avoid repeating same string few times